django_slack/api.py

```diff
@@ -51,6 +51,11 @@ def slack_message(template, context=None, attachments=None, fail_silently=None,
             'render': ALWAYS,
             'required': NOT_REQUIRED,
         },
+        'as_user': {
+            'default': app_settings.AS_USER,
```

Comment:
Missing the corresponding entry in ``AppSettings``, no?

(Does tihs code work for you...?)